Tools/crmpak/main.cpp

```diff
@@ -153,7 +153,7 @@ int main(int argc, char *argv[])
     // Open the room, export list of block ids ('l' command).
     //-----------------------------------------------------------------------//
     RoomDataSource datasrc;
-    HError err = OpenRoomFile(in_roomfile, datasrc);
+    HError err = static_cast<PError>(OpenRoomFile(in_roomfile, datasrc));
```

Comment:
Why is this fix necessary, but there's no error reported by any of the build on CI?

Reply:
The tools aren't being built in the CI currently. I thought it would only be necessary to build when there's a plan to how they are going to be released. If you want I can turn on for the CMake builds.

Reply:
So, this fails on one of the non-msvs compilers, like gcc?

Reply:
It failed to me on Clang, you can probably use it even on Windows.

Reply:
I don't understand why you seem to want an HError but made a function that returns something else.